src/components/backend-ai-signup.ts

```diff
@@ -199,47 +218,72 @@ export default class BackendAiSignup extends BackendAIPage {
     //this.shadowRoot.querySelector('#signup-button').setAttribute('disabled', 'true');
   }
 
+  _clearUserInput() {
+    this._toggleInputField(true);
+    const inputFields: Array<string> = ['#id_user_email', '#id_user_name', '#id_token', '#id_password1', '#id_password2'];
+    inputFields.map((el: string) => {
+      this.shadowRoot.querySelector(el).value = "";
```

Comment:
Minor issue, but please be consistent with the quotes. Since all other strings in the method uses a single quote(`'`), replace double quote(`"`) here.